### DISCUSSION TITLE
Análise de métricas e estatísticas em UX

### DISCUSSION THREAD

**Discussion by Cleysonlb** [frontendbr/forum#483] [Perguntas e Respostas]
Falae galera tudo bem?! Gostaria de trazer essa discussão aqui pro fórum, como vocês analisam os dados e métricas retirados de pesquisas em UX?

**Comment by felipefialho** [comment]
@matmarsiglio Teve uma discussão da hora disso no FrontUX uma vez. Consegue marcar umas pessoas aqui? :)

**Comment by mtmr0x** [comment]
@LFeh acho que os nomes foram mais designers e sem GitHub :( Nao consigo lembrar de alguem que achei no GitHub aqui. Recomendo entrar no grupo do Slack UXSP. http://uxsp.in/

**Comment by kvnol** [comment]
@mjcoffeeholick

**Comment by nandomoreirame** [comment]
:+1:
